app/containers/SignIn/index.js

```diff
@@ -191,33 +189,31 @@ class SignIn extends Component {
         withMove={false}
         fromBackgroundStyle={styles.wrapperToBackground}
       >
-        <StatusBar
-          backgroundColor={theme.PRIMARY_STATUS_BAR_COLOR} />
-        <ScrollView
-          keyboardDismissMode={'on-drag'}
-          style={styles.container}>
+        <StatusBar backgroundColor={theme.PRIMARY_STATUS_BAR_COLOR} />
```

Comment:
Please stick to the coding styles.
The first property should and the component should not be in the same line. (Make the necessary changes in all the places)

```suggestion
        <StatusBar
           backgroundColor={theme.PRIMARY_STATUS_BAR_COLOR} />
```